From b38994a5741ed9d75211b6ac47753af6cde70f99 Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Fri, 20 Sep 2019 17:01:08 +0100
Subject: [PATCH] Minimum CMake version of 3.6.0 [METV-2497]

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8d1d640..925d02f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required( VERSION 2.8.11 FATAL_ERROR )
+cmake_minimum_required( VERSION 3.6.0 FATAL_ERROR )
 
 include( ${INCLUDE_CONFIG}-local-config.cmake OPTIONAL )
 
-- 
2.13.7

